R/LearnerClassifDebug.R

```diff
@@ -57,29 +58,28 @@ LearnerClassifDebug = R6Class("LearnerClassifDebug", inherit = LearnerClassif,
         feature_types = c("logical", "integer", "numeric", "character", "factor", "ordered"),
         predict_types = c("response", "prob"),
         param_set = ps(
-          error_predict        = p_dbl(0, 1, default = 0, tags = "predict"),
-          error_train          = p_dbl(0, 1, default = 0, tags = "train"),
-          message_predict      = p_dbl(0, 1, default = 0, tags = "predict"),
-          message_train        = p_dbl(0, 1, default = 0, tags = "train"),
-          predict_missing      = p_dbl(0, 1, default = 0, tags = "predict"),
+          error_predict = p_dbl(0, 1, default = 0, tags = "predict"),
+          error_train = p_dbl(0, 1, default = 0, tags = "train"),
+          message_predict = p_dbl(0, 1, default = 0, tags = "predict"),
+          message_train = p_dbl(0, 1, default = 0, tags = "train"),
+          predict_missing = p_dbl(0, 1, default = 0, tags = "predict"),
```

Comment:
@lorenzwalthert 

IIRC we had support for this in the old fork - was there a specific transformer or did we do a hacky island solution here?

Reply:
alignment is supported in general with {styler}, but only with right-alignment:

``` r
styler.mlr::style_text(
  'call(
  x  =   2, 
  uz = 333
  )
  '
)
#> call(
#>   x  =   2,
#>   uz = 333
#> )
```

<sup>Created on 2021-04-13 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

Reply:
I suggest to manually align them to one of the supported alignments in https://styler.r-lib.org/articles/detect-alignment.html to make sure styler does not *destroy* the formatting of deliberately aligned code.

Reply:
Left alignment support on the way: https://github.com/r-lib/styler/pull/774

Reply:
great 🎉

Reply:
You need to have the devel version of styler with `9f9bdff` at least yes. I am adding a minimal version requirement to {styler.mlr} aud bump the devel version in {styler}.

Reply:
Done. I suggest to re-style.

Reply:
👍 

resetted and restyled - some more alignment takes it seems :)